app/app.go

```diff
@@ -244,6 +266,80 @@ func New(
 	return app, nil
 }
 
+func (app *App) initIBC() error {
+	// set up non depinject support modules store keys
+	if err := app.RegisterStores(
+		storetypes.NewKVStoreKey(ibcexported.StoreKey),
+		storetypes.NewKVStoreKey(ibctransfertypes.StoreKey),
+		storetypes.NewTransientStoreKey(paramstypes.TStoreKey),
```

Comment:
What is this transient key being used for?

Reply:
Gotta confess that I don't know - I picked it up from the original v9 integration that was removed: https://github.com/thezkcloud/zkcloud/commit/8473523a0babd7c4f46999c56b91f2973aab60ff#diff-e5871b24bdf6b577029025d3aa359c4a8fc6d4370b5442ca8a015dcfb22cf81eL51

That sounds like not needed so I'll drop it.